### PR TITLE
Re #11, make important client column info easier to see

### PR DESCRIPTION
**Pull request by plocket**
Actually, it is at a point where I think it could do some good, so maybe worth making a full PR to at least have this version up.

Addresses #11 partially.

1. Turns dates into number of days. Unable to do business days currently.
2. Keeps the actual dates around as hover text (though it's just HTML `title`, so takes a bit of time hovering to appear)
2. Reduces some padding
3. Adds text of client id and organization name into the 'names' column and then moves the id and org columns to the end of the table.

### CODE REVIEW

**Comment by plocket** [comment]
User feedback
Columns:
- Most important info from left to right
   - Red dot
   - Name/other info
   - Status alone (talk a bit more a bit later)
   - Assignee [including year date]
   - Dates
   - Language
- **Move** assign button container (move whole cell to help with bookmarklet additional button?), but want keep the date with it somehow (maybe in button?)
- **Copy** filling status

Not as relevant columns:
- Unemployment Income
- State
- Certification
- 'Drop off' - you can use filters to keep those out

Misc todos:
- Test this on the assigned clients page too.

Roses:
- All that info in client name is a good way to condense info.

Thorns:
- Do not change headings for day columns as they're used in documentation. (Maybe just break up with new lines to keep width under control?)

Other notes:
- 'Updated at' is useful once you've picked up a client/assigned yourself to them, though not otherwise.
   - Filer not responded vs. volunteer not responded? Possibly not a useful calculation. Can't calculate the the things we need to know just based on the info present on the client page.
- Actions when looking at 'All clients':
   - Set filters.
   - Look for red dots first with those clients at the top.
- ID: not super important to be able to copy ID on the client page - usually copied within the client ticket.

**Comment by plocket** [comment]
This messes up a couple other bookmarklets because they're using indexes to find the columns. I'll work on fixing those before we merge this in.

File: squish.user.js [hunk 1]
```diff
@@ -0,0 +1,289 @@
...
+// require_script.src = 'scripts/require.js';
+// document.body.appendChild( require_script );
+// let moment_script = document.createElement('<script src="moment.js"></script>');
+// let days_script = document.createElement('<script src="moment-business-days.js"></script>');
+
+// let business_days = require('moment-business-days');
+
+var squish = function () {
+
+  // Prevent stuff from being added multiple times (implement after development)
```
Comment (BryceStevenWilley) [inline squish.user.js]:
```suggestion
  // Prevent stuff from being added multiple times
```

File: squish.user.js [hunk 2]
```diff
@@ -0,0 +1,289 @@
...
+  if ( document.querySelector( '.squish' )  ) {
+    console.log( 'squish has already run' );
+    return;
+  }
+
+  document.body.className += ' squish';
```
Comment (BryceStevenWilley) [inline squish.user.js:30]: If this is on the whole body, maybe make it more specific, i.e. `bookmarklet-squish`.

File: squish.user.js [hunk 3]
```diff
@@ -0,0 +1,289 @@
...
+
+  // // Be able to turn dates into business days
+  // let moment_script = document.createElement('script');
+  // moment_script.src = 'moment.js';
+  // document.body.appendChild( moment_script );
+  // let days_script = document.createElement('script');
+  // days_script.src = 'moment-business-days.js';
+  // document.body.appendChild( days_script );
```
Comment (BryceStevenWilley) [inline squish.user.js:38]: Should change this to be `TODO: the feature hasn't been implemented because of XYZ`

File: squish.user.js [hunk 4]
```diff
@@ -0,0 +1,291 @@
...
+  let name_header_index = 1;
+  let id_index = 2;
+  let org_header_index = 3;
+  let lang_header_index = 4;
+  let unemployment_index = 5;
+  let updated_index = 6;
+  let waiting_index = 7;
+  let created_index = 8;
+  for ( let header_i = 0; header_i < column_headers.length; header_i++ ) {
+    let text = column_headers[ header_i ].innerText;
```
Comment (BryceStevenWilley) [inline squish.user.js:61]: Should probably add
```suggestion
    let text = column_headers[ header_i ].innerText.toLowerCase().trim();
```

File: squish.user.js [hunk 5]
```diff
@@ -0,0 +1,291 @@
...
+      updated_days = Math.round(( today - date) / one_day_ms);
+    }
+    if ( waiting_text ) {
+      let date = new Date( waiting_text + ` ${this_year}` );
+      waiting_days = Math.round(( today - date) / one_day_ms);
+    }
+    if ( created_text ) {
+      let date = new Date( created_text + ` ${this_year}` );
+      created_days = Math.round(( today - date) / one_day_ms);
+    }
```
Comment (BryceStevenWilley) [inline squish.user.js:122]: Minor, and optional: these 3 calls would probably be clearer (and easier to maintain) as a function, called 3 times.

**Review by BryceStevenWilley — APPROVED** [review]
LGTM! Some minor nits

File: squish.user.js (hunk 3 above)
Comment (plocket) [reply squish.user.js:38]: Made #29

File: squish.user.js (hunk 4 above)
Comment (plocket) [reply squish.user.js:61]: Can't commit this as the logic would have to be changed as well, but good suggestion. [Made #30]

File: squish.user.js (hunk 5 above)
Comment (plocket) [reply squish.user.js:122]: Created #31

File: squish.user.js (hunk 2 above)
Comment (plocket) [reply squish.user.js:30]: I'll try to remember this, but it'll be lower on my priority list in this particular context.

**Pull request merged**